.eslintrc.js

```diff
@@ -63,7 +63,7 @@ module.exports = {
     },
     {
       files: ['test-processor.js', 'test-setup.js', 'test/**'],
```

Comment:
Test folder is ignored in
`eslintignore`, do you know is it was added intentionally?

Reply:
No, seems weird since there was a plugin to test that stuff 😄 
I'll try un-ignoring and see if there are issues